### PR TITLE
Add eBay to adopters.yaml

### PR DESCRIPTION
**Pull request by svrnm**
@vjsamuel, is it ok to put you here as a contact? We try to have someone to reach out to in cases where we change this list or need additional infos.

Also feel free to add to the list of components (e.g. language SDKs) that eBay is using.

TY!

### CODE REVIEW

File: data/ecosystem/adopters.yaml [hunk 1]
```diff
@@ -14,6 +14,11 @@
   components: [Collector, Python]
   reference: ''
   contact: ''
+- name: eBay
+  url: https://www.ebay.com
+  components: [Collector]
+  reference: /blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/
+  contact: 'https://github.com/vjsamuel'
```
Comment (chalin) [inline data/ecosystem/adopters.yaml:21]: I don't think that you need quotes here:
```suggestion
  contact: https://github.com/vjsamuel
```

Comment (chalin) [inline data/ecosystem/adopters.yaml:21]: Nit: I don't think that you need quotes here:

```suggestion
  contact: https://github.com/vjsamuel
```

File: data/ecosystem/adopters.yaml [hunk 2]
```diff
@@ -14,6 +14,11 @@
   components: [Collector, Python]
   reference: ''
   contact: ''
+- name: eBay
+  url: https://www.ebay.com
+  components: [Collector]
+  reference: /blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/
```
Comment (chalin) [inline data/ecosystem/adopters.yaml:20]:
```suggestion
  reference: /blog/2022/why-and-how-ebay-pivoted-to-opentelemetry/ # cspell:disable-line
```

File: data/ecosystem/adopters.yaml [hunk 3]
```diff
@@ -14,6 +14,11 @@
   components: [Collector, Python]
   reference: ''
   contact: ''
+- name: eBay
+  url: https://www.ebay.com
+  components: [Collector]
```
Comment (vjsamuel) [inline data/ecosystem/adopters.yaml:19]:
```suggestion
  components: [Collector, Java, Javascript, Go]
```

**Review by vjsamuel — COMMENTED** [review]
only a minor change to cover the SDKs we use (so far). the contact information works for me.

**Review by cartermp — APPROVED** [review]

**Pull request merged**